src/screens/ProfileScreen/elements/KarmaLock.tsx

```diff
@@ -45,26 +45,28 @@ export const KarmaLock = (props: {onPressCreatePost: () => void}) => {
           contentStyle={{
             borderRadius: 10,
             padding: 16,
-            height: 'auto'
+            height: 'auto',
+            backgroundColor: COLORS.almostBlack
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `height` property in the `contentStyle` object has been explicitly set to `'auto'`, and the `backgroundColor` property has been added. These changes could improve the layout and appearance of the component.
```diff
- height: 'auto'
+ height: 'auto',
+ backgroundColor: COLORS.almostBlack
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->